thinc/model.py

```diff
@@ -299,13 +299,17 @@ def finish_update(self, optimizer: Optimizer) -> None:
         with each parameter and gradient of the model.
         """
         for node in self.walk():
+            orig_ops = node.ops
             for name in node.param_names:
                 if node.has_grad(name):
                     param = node.get_param(name)
                     grad = node.get_grad(name)
+                    if hasattr(optimizer,'ops'):
+                        param = optimizer.ops.xp.asarray(param)
+                        optimizer.ops.xp.asarray(grad)
```

Comment:
Why `ops.xp.asarray` and not `os.asarray`?

Reply:
I think that was wrong ...